docs/application/native/guides/ui/overview.md

```diff
@@ -23,6 +23,10 @@ You can use the following user interface features in your native applications:
   - Homescreens / launch pads
   - Advanced watch faces for wearable devices
 
+- [Tizen Window System Shell (TZSH)](tizen-ws-shell.md)
+
+  You can use the Tizen window system shell if you are manipulating windows of the sytem graphical user interfaces (GUI) services such as the quickpanel. It allows developers to control windows of system GUI services and be notified state changes of them within an application. Most applications don't need to use the tzsh's functionalities. However, some applications may want to perform manipulation of system GUI service's window in certain situations.
```

Comment:
You can use the Tizen Window System Shell(TZSH) to manipulate the windows component of the system graphical user interfaces (GUI) services such as the quick-panel. It allows you to control the windows of system GUI services and and get notifications about the state changes of each window within an application. Not all applications require to use the TZSH's functionalities. However, in some cases, certain applications may require to perform manipulation of system GUI service's window.

Reply:
Thanks for comments, @nawab-samsung and @mijin85cho.

windows component of the system graphical user interfaces -> windows of ~~~

I'd like to remove a word 'component' from 'windows component'. Usually, the word 'window' is used as itself individually.

quick-panel -> quickpanel

The quickpanel is a proper noun in the Tizen. Not hyphenated compound style 'quick-panel' but closed compound style 'quickpanel' is already used here and there from several years ago. So I just want to keep using 'quickpanel' without hyphen.


I modified introduction sentences according to your comments as follows:


You can use the TZSH to manipulate the windows of the system graphical user interfaces (GUI) services such as the quickpanel. It allows you to control the windows of system GUI services and get notifications about the state changes of each window within an application. Not all applications require to use the TZSH's functionalities. However, in some cases, certain applications may require to perform manipulation of system GUI service's window.

Reply:
@gl-lee  fine